src/panels/lovelace/editor/find-card-documentation-url.ts

```diff
@@ -0,0 +1,14 @@
+import { coreCards } from "./lovelace-cards";
+import {
+  getCustomCardEntry,
+  customCards,
+} from "../../../data/lovelace_custom_cards";
+
+export const findCardDocumentationURL = (type: string): string | undefined => {
+  const coreCard = coreCards.find((c) => c.type === type);
```

Comment:
Core cards can just be?
```js
`https://www.home-assistant.io/lovelace/{$type}`
```

Reply:
We should identify a custom card by the `custom:` prefix

Reply:
I have that at first but didn't want to hard code them if something changed to be always the type at the end

Reply:
But makes sense